crates/cli-support/src/js/outgoing.rs

```diff
@@ -130,6 +130,27 @@ impl<'a, 'b> Outgoing<'a, 'b> {
                 Ok(format!("v{}", i))
             }
 
+            NonstandardOutgoing::CachedString {
+                offset,
+                length,
+                owned,
+            } => {
+                let ptr = self.arg(*offset);
+                let len = self.arg(*length);
+                let tmp = self.js.tmp();
+
+                self.js.typescript_optional("string");
```

Comment:
I think this needs to be `typescript_required` because this is a required string argument, not an optional one

Reply:
Ah, good catch, I need to pass in a boolean indicating whether it's required or not.